com.insynchq.Insync.appdata.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.insynchq.Insync</id>
+  <name>Insync</name>
+  <project_license>LicenseRef-proprietary</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <developer_name>Insync</developer_name>
+  <summary>Google Drive, OneDrive and Dropbox syncing</summary>
+  <url type="homepage">https://www.insynchq.com/</url>
+  <description>
+    <p>
+      <em>
+      You will need a paid license to use Insync.
+      </em>
+    </p>
+    <p>
+    Manage your Google Drive, OneDrive, and Dropbox files straight from your Desktop
+    </p>
+    <p>
+    With Insync, you can now sync, edit, and share files without opening a million browser tabs
+    </p>
```

Comment:
I think this still require the usual disclaimer that it is not official, blah blah.

Reply:
Added